### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by raravena80**
Adding the logo. Will update once we get a better transparent background logo.

### CODE REVIEW

**Comment by k82cn** [comment]
Cool!

/lgtm
/approve

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<img width="200" alt="Tag-runtime" src="https://user-images.githubusercontent.com/7659560/160054856-5b05fa62-c5f0-4483-b74d-97e080fe6fd7.png">
```
Comment (kad) [inline README.md:1]: Maybe better to put image into this repo and reference it directly within one repo?

Comment (raravena80) [reply README.md:1]: Not a fan of storing images in a repo, but any suggestions about what directory to store these under?

Comment (raravena80) [reply README.md:1]: oh, that works!

**Review by kad — COMMENTED** [review]
/lgtm

**Pull request merged**